Fill.py

```diff
@@ -466,7 +467,7 @@ def mark_for_locking(location: Location):
 
     inaccessible_location_rules(multiworld, multiworld.state, defaultlocations)
 
-    remaining_fill(multiworld, excludedlocations, filleritempool)
+    remaining_fill(multiworld, excludedlocations, filleritempool, name="Remaining Excluded")
```

Comment:
think i'd prefer not using the kwarg

Reply:
Actually, looking at the "Files changed" i would have made the opposite suggestion just now :D
I think passing arguments that are not part of the main functionality, especially when they have a default, is good as kwarg. Either is fine here though.